### ISSUE TITLE
Images + texts for the new landing page on themeisle

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/zerif-lite#410]
We need to replace this http://themeisle.com/themes/zerif-lite/ with the new landing page.

**Comment by abaicus** [comment]
[zerif-lite-lp-pics.zip](https://github.com/Codeinwp/zerif-lite/files/471994/zerif-lite-lp-pics.zip)

@rodica-andronache

**Comment by rodica-andronache** [comment]
- [x] images from andrei
- [x] texts from adelina
- [ ] reviews from mariusg?
- [ ] update ti page

**Comment by rodica-andronache** [comment]
Reviews:

- Awesome theme & fast support!
My first website ever but the site looks great!
Great customizability, fast support on the forum and super cool parallax effect!
Cheers!

- On point with functionality.
Normally I don't review themes. But this theme absolutely deserves it. Unlike most .org themes, Zerif actually has the user in mind and ( at least it seems so far ) all the functionality mapped out.

You remove an image in one of the sections, the section disappears like you'd expect. If you remove all the widgets from the footer, the rest of the footer ACTUALLY spreads out...

It's honestly the little things and the attention to detail that has blown me away about this theme. I'm absolutely blown away by this theme.
https://profiles.wordpress.org/phyrax/

- Nice for persian
It very nice theme for RTL languages like Persian .
Also very easy to use.
Thanks

- Rock solid. Awesome for free & also totally worth the Pro upgrade.
I started with Zerif Lite which is the free version of the Pro alternative. Setting up a site is very easy and intuitive. The design is clean and works on all computers and devices. I eventually upgraded to the Pro version because I run my business on this theme and wanted all the customization and support features. The support is great. The ThemeIsle team can’t, and shouldn’t be expected to, totally customize the theme for every person but they are very helpful with a lot of basic customizations to personalize the theme even more than what’s offered up front.

I’ve been really happy from day 1 and I recommend it to all my friends and colleagues. The only reason not to go with this theme is if the design simply doesn’t suit your vision. Otherwise it’s fantastic.
